crates/utils/src/email.rs

```diff
@@ -78,7 +78,12 @@ pub fn send_email(
   };
 
   // Set the creds if they exist
-  if let (Some(username), Some(password)) = (email_config.smtp_login, email_config.smtp_password) {
+  let smtp_password = match std::env::var("LEMMY_SMTP_PASSWORD") {
```

Comment:
Seems fine, but at some point these env var additions are going to get a bit out of hand. What's the security differenence between just not checking in your entire `lemmy.hjson` config file, which has other secrets anyway, and an env var?

Reply:
My lemmy.hjson doesn't have any secrets, I load the DB url from an env var.
I can't not check the file in, because... nix. It has to be checked in to get picked up.

Also, I just simply want the file checked in because it can contain some useful information to other NixOS users on how to set Lemmy up.

It's all a matter of convenience - as I said in the original issue.